### ISSUE TITLE
if left bar is collapsed then there's no explore button and rooms+ only handles creating

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#10910]

**Comment by aaronraimist** [comment]
See https://matrix.to/#/!YTvKGNlinIzlkMTVRl:matrix.org/$P3kTn-UaQqWgUx8IuxiT7Zad_4pBNS4x99O9F4b6I1Q?via=matrix.org&via=privacytools.io&via=chat.weho.st for one users confusion about this
![image](https://user-images.githubusercontent.com/5855073/65400351-476a8f00-dd87-11e9-9af3-20905cfc791b.png)

The explore button probably should be visible at the top but Riot also probably should be more aggressive about collapsing the user list first, before the room list. Riot should also probably collapse the community panel on small screens, especially if you have no communities or tags. 

Related: #8042 #5811

**Comment by DJCrashdummy** [comment]
IMHO the best idea would be to go back having a combined add-/create-function behind the `+`-sign, and if it is really necessary an additional separate (perhaps optional?) explore-button.

**Comment by BrenBarn** [comment]
I don't really understand why the button isn't shown.  If the window is too narrow, why can't the explore button be shown anyway, stacked on top the way all the others are?

**Comment by BlueCatYellowCat** [comment]
I spent a good 10 minutes trying to join a room because I had opened riot.im with a browser window less than 1000px wide.
Exploring to find existing rooms is one of the most important feature for new members so I think this UI/UX issue is highly important!

**Comment by vith** [comment]
First time using the new design. I tried for 10-15 minutes looking everywhere and never figured it out.

The sidebar is collapsed by default on a fresh install of the desktop app like this:

![image](https://user-images.githubusercontent.com/3265539/71825017-8aafeb80-3060-11ea-85cb-7030590d09f2.png)

There seems to be no hint that it even *is* collapsed. I didn't guess that there was an expanded mode.

Eventually I started googling for how to list/join public rooms, but didn't learn what I was missing until I found this issue.
